sns_ik_examples/README.md

```diff
@@ -0,0 +1,41 @@
+# README:  sns_ik_examples
+
+This directory contains scripts that run some tests of the SNS-IK library using
+the Sawyer and Baxter robot models. There are three scripts:
+- `test_baxter.launch` compares the various IK solvers using the Baxter robot model
+- `test_sawyer.launch` compares the various IK solvers using the Sawyer robot model
+- `test_ik_solvers.launch` compares the various IK solvers on large random data sets
```

Comment:
It would be helpful to mention some of the testing options in test_ik_solvers.launch. I can go over them with you in person.

Reply:
I added some documentation to the launch file itself, as well as in the readme.